### PR TITLE
Christmas

### PR DESCRIPTION
**Pull request by Simson475**
Jule tema til stregsystemet

### CODE REVIEW

**Review by Zaph-x — APPROVED** [review]

**Review by falkecarlsen — APPROVED** [review]
:shipit: JIT development for Christmas styling

File: stregsystem/static/stregsystem/snow.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    left: 42%;
+    -webkit-animation-delay: 7s, 4s;
+    animation-delay: 5.69s, 7.5s
+}
+
+.snowflake:nth-of-type(23) {
+    left: 69%;
+    -webkit-animation-delay: 2.5s, 4s;
+    animation-delay: 42s, 12.9s
+}
```
Comment (SEbbaDK) [inline stregsystem/static/stregsystem/snow.css:209]: I hate it!
:+1:

**Review by SEbbaDK — APPROVED** [review]
I kinda miss the mysterious CDN we were using before, but it will do xD

File: stregsystem/static/stregsystem/snow.css (hunk 1 above)
Comment (falkecarlsen) [reply stregsystem/static/stregsystem/snow.css:209]: Agreed :ok_hand:

**Pull request merged**